third_party/accessibility/ax/platform/ax_platform_atk_hyperlink.cc

```diff
@@ -0,0 +1,279 @@
+}
+
+static gboolean ax_platform_atk_hyperlink_do_action(AtkAction* action,
+                                                    gint index) {
+  g_return_val_if_fail(ATK_IS_ACTION(action), FALSE);
+  g_return_val_if_fail(!index, FALSE);
+
+  AXPlatformNodeAuraLinux* obj =
+      ToAXPlatformNodeAuraLinuxFromHyperlinkAction(action);
+  if (!obj)
+    return FALSE;
+
+  obj->DoDefaultAction();
+
+  return TRUE;
+}
+
+static gint ax_platform_atk_hyperlink_get_n_actions(AtkAction* action) {
+  g_return_val_if_fail(ATK_IS_ACTION(action), FALSE);
+
+  AXPlatformNodeAuraLinux* obj =
+      ToAXPlatformNodeAuraLinuxFromHyperlinkAction(action);
+  if (!obj)
+    return 0;
+
+  return 1;
+}
+
+static const gchar* ax_platform_atk_hyperlink_get_description(AtkAction* action,
+                                                              gint index) {
+  g_return_val_if_fail(ATK_IS_ACTION(action), FALSE);
+  g_return_val_if_fail(!index, FALSE);
+
+  AXPlatformNodeAuraLinux* obj =
+      ToAXPlatformNodeAuraLinuxFromHyperlinkAction(action);
+  if (!obj)
+    return nullptr;
+
+  // Not implemented
+  return nullptr;
+}
+
+static const gchar* ax_platform_atk_hyperlink_get_keybinding(AtkAction* action,
+                                                             gint index) {
+  g_return_val_if_fail(ATK_IS_ACTION(action), FALSE);
+  g_return_val_if_fail(!index, FALSE);
+
+  AXPlatformNodeAuraLinux* obj =
+      ToAXPlatformNodeAuraLinuxFromHyperlinkAction(action);
+  if (!obj)
+    return nullptr;
+
+  return obj->GetStringAttribute(ax::mojom::StringAttribute::kAccessKey)
+      .c_str();
+}
+
+static const gchar* ax_platform_atk_hyperlink_get_name(AtkAction* atk_action,
+                                                       gint index) {
+  g_return_val_if_fail(ATK_IS_ACTION(atk_action), FALSE);
+  g_return_val_if_fail(!index, FALSE);
+
+  AXPlatformNodeAuraLinux* obj =
+      ToAXPlatformNodeAuraLinuxFromHyperlinkAction(atk_action);
+  if (!obj)
+    return nullptr;
+
+  int action;
+  if (!obj->GetIntAttribute(ax::mojom::IntAttribute::kDefaultActionVerb,
+                            &action))
+    return nullptr;
+  std::string action_verb =
+      ui::ToString(static_cast<ax::mojom::DefaultActionVerb>(action));
+  ATK_AURALINUX_RETURN_STRING(action_verb);
+}
+/*
+static const gchar* ax_platform_atk_hyperlink_get_localized_name(
+    AtkAction* atk_action,
+    gint index) {
+  g_return_val_if_fail(ATK_IS_ACTION(atk_action), FALSE);
+  g_return_val_if_fail(!index, FALSE);
+
+  AXPlatformNodeAuraLinux* obj =
+      ToAXPlatformNodeAuraLinuxFromHyperlinkAction(atk_action);
+  if (!obj)
+    return nullptr;
+
+  int action;
+  if (!obj->GetIntAttribute(ax::mojom::IntAttribute::kDefaultActionVerb,
+                            &action))
+    return nullptr;
+  std::string action_verb =
+      ui::ToLocalizedString(static_cast<ax::mojom::DefaultActionVerb>(action));
+  ATK_AURALINUX_RETURN_STRING(action_verb);
+}*/
```

Comment:
Why has this been commented out?

Reply:
I commented this code out in order to fix the build error, `ui::ToLocalizedString` should use `l10n` in Chromium's `base/l10n`.
Screen reader use `get_name` and `get_description` to read information usually. It's no matter to commented this code out.

```
static void atk_action_interface_init(AtkActionIface* iface) {
  iface->do_action = ax_platform_atk_hyperlink_do_action;
  iface->get_n_actions = ax_platform_atk_hyperlink_get_n_actions;
  iface->get_description = ax_platform_atk_hyperlink_get_description;
  iface->get_keybinding = ax_platform_atk_hyperlink_get_keybinding;
  iface->get_name = ax_platform_atk_hyperlink_get_name;
  // iface->get_localized_name = ax_platform_atk_hyperlink_get_localized_name;
}
```

Reply:
Okay, thanks. I can also see the build error when the code is uncommented.